iron-overlay-manager.html

```diff
@@ -43,8 +43,8 @@
     // NOTE: Use useCapture=true to avoid accidentally prevention of the closing
     // of an overlay via event.stopPropagation(). The only way to prevent
     // closing of an overlay should be through its APIs.
-    Polymer.Gestures.add(document, 'tap', null);
-    document.addEventListener('tap', this._onCaptureClick.bind(this), true);
+    Polymer.Gestures.add(document.documentElement, 'tap', null);
```

Comment:
This is the only line that needs to change, as we just need to enable the Polymer Gestures to recognize `tap`. It would be great to add also a comment on why we do this & point to the polymer issue, e.g.
> // NOTE: enable tap on `<html>` to workaround Polymer/polymer#4459

I just want to avoid any possible regressions, e.g. someone might be listening for the same event with useCapture = true and stop the propagation:
```js
document.addEventListener('tap', function(e) {
  e.stopPropagation();
}, true)
```
If we listen for `tap` on `document.documentElement`, we wouldn't capture it.